pkgs/top-level/release.nix

```diff
@@ -163,44 +163,33 @@ let
         };
 
       stdenvBootstrapTools = with lib;
-        genAttrs systemsWithAnySupport
-          (system: {
-            inherit
-              (import ../stdenv/linux/make-bootstrap-tools.nix {
+        genAttrs systemsWithAnySupport (system:
+          if hasSuffix "-linux" system then
```

Comment:
Can we use stdenv.isLinux?

Reply:
No, this is iterating over `systemsWithAnySupport`.